management/backup/locations_test.go

```diff
@@ -203,6 +203,166 @@ func TestListLocations(t *testing.T) {
 	assert.True(t, found, "Expected location not found")
 }
 
+func TestUpdateLocation(t *testing.T) {
```

Comment:
let's preserve naming consistency
```suggestion
func TestChangeLocation(t *testing.T) {
```